macros/get_monthly_ad_analytics_by_member_region_columns.sql

```diff
@@ -0,0 +1,20 @@
+{% macro get_monthly_ad_analytics_by_member_region_columns() %}
+
+{% set columns = [
+    {"name": "campaign_id", "datatype": dbt.type_int()},
+    {"name": "month", "datatype": dbt.type_string(), "alias": "date_month"},
+    {"name": "clicks", "datatype": dbt.type_int()},
+    {"name": "member_region", "datatype": dbt.type_int()},
+    {"name": "impressions", "datatype": dbt.type_int()},
+    {"name": "conversion_value_in_local_currency", "datatype": dbt.type_numeric()},
+    {"name": "cost_in_local_currency", "datatype": dbt.type_numeric()},
+    {"name": "cost_in_usd", "datatype": dbt.type_numeric()}
+] %}
+
+{{ fivetran_utils.add_pass_through_columns(columns, var('linkedin_ads__conversion_fields')) }}
```

Comment:
I believe this is an artifact for backwards compatibility, so not sure if this macro is needed for a net new model..